### ISSUE TITLE
No update notifications for UnifiedNlp

### ISSUE THREAD

**Issue by Nutomic** [microg/microg.org#10]
I installed UnifiedNlp via the microg F-Droid repo. For some reason i'm never getting update notifications for UnifiedNlp updates. Notifications work fine for apps from the main F-Droid repo (including automatic download).

F-Droid version: 0.102
Android version: Cyanogenmod 14.1

Edit: F-Droid also tells me that 0.2.4-39 is signed with a different key than 0.2.4-30. Is this intended?

**Comment by JonnyTech** [comment]
I got that too. Did confuse me. Had to uninstall build 30 then install build 39. Assumed that the different key was to blame. Was it a special build not intended for general use? Or should I report it to the F-Droid devs?

**Comment by ArchangeGabriel** [comment]
@JonnyTech Yes, different key means need to reinstall. I can’t say why the key chandeg, but two things I have in mind regarding the other points:

1) -* versions are previews, it’s normal to not get notifications about those. The next notification should be for 0.2.5-0 (or -1).

2) I’m not sure about this, but your version number sounds like microG GMS Core one. This could be because of microG repo, don’t know about this, but UnifiedNlp in F-Droid is 1.6.7.

**Comment by JonnyTech** [comment]
Yes, latest version of F-Droid (v0.102) with "Expert Mode >> Unstable Updates" enabled, which states "Suggest updates to unstable versions". It is definitely UnifiedNlp, see screenshot attached:

![02439](https://cloud.githubusercontent.com/assets/3584732/21520123/f7d16870-cce9-11e6-881a-583997f205bf.png)
